src/components/shared/basic/ReleaseProcess/AppMarketCard/index.tsx

```diff
@@ -450,6 +451,23 @@ export default function AppMarketCard() {
     await updateDocumentUpload(data).unwrap()
   }
 
+  const handleDownload = async (documentName: string, documentId: string) => {
+    if (fetchDocumentById)
+      try {
+        const response = await fetchDocumentById({
+          appId,
+          documentId,
+        }).unwrap()
+
+        const fileType = response.headers.get('content-type')
```

Comment:
these two lines are duplicated in many place. Can you move this to utils?

Reply:
Done